static/app/views/alerts/list/rules/row.tsx

```diff
@@ -55,6 +65,7 @@ function RuleListRow({
   userTeams,
```

Comment:
whats the difference between userTeams and filteredTeams?

Reply:
Ah my mistake - no difference, I was trying another approach earlier and forgot `userTeams` had what I needed. Updating it back to use `userTeams`